site/docs/v1/tech/premium-features/scim/okta.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: doc
+title: Okta SCIM Client
+description: Configure Okta as a SCIM client
+navcategory: premium
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+This is a basic example to get started configuring Okta as a SCIM client and establish a connection between FusionAuth and Okta for provisioning purposes.
+
+For complete documentation, please refer to the Okta provisioning documentation.
+
+To configure the integration, in your Okta Dashboard, find your Application configured for provisioning and click on the *Provisioning* tab of the Application configuration.
+
+Select *Integration* from the left *Settings* navigation menu, and find the required configuration fields.
+
+image::okta-scim-client-credentials.png[Okta SCIM Provisioning Integration,width=1200,role=shadowed]
```

Comment:
We don't need 'role=shadowed' do we?

Reply:
I thought it still did something, but I'll confirm and remove if it is dead. Thanks.